src/isar/state_machine/states/monitor.py

```diff
@@ -85,6 +85,11 @@ def _queue_inspections_for_upload(self, current_step: InspectionStep):
             )
             return
 
+        if len(inspections) == 0:
```

Comment:
```suggestion
        if not inspections:
```

Reply:
More the "python" way of doing it I think.

Reply:
Good point, fixed.